src/login/logind-action.c

```diff
@@ -346,6 +389,13 @@ int manager_handle_action(
                 return 1;
         }
 
+        if (handle == HANDLE_SECURE_ATTENTION_KEY) {
+                if (!is_edge)
+                        return 0;
```

Comment:
you have exactly this check in manager_handle_action_secure_attention_key(), hence you can drop it here.